dev/io.openliberty.servlet.internal.6.0.factories/bnd.bnd

```diff
@@ -0,0 +1,104 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2022 IBM Corporation and others.
```

Comment:
if this is new file, I think we only need 2022 ?

Reply:
Sorry about that- I had copied this project from the 4.0.factories project and inherited that.  I'll get the old copyright year removed here and in the other place that you identified.